chore(codegen): skip stacktrace tests on big endian test workflow

sapphi-red opened the pull request:
This PR tries to fix the following test failure.
https://github.com/oxc-project/oxc/actions/runs/17313194902/job/49151067079#step:5:325

This failure is probably happening because node binary is missing in the docker image that the test is running. While we can add the node binary in the docker container, I think we can skip the test as I guess there aren't any endian-specific code.

sapphi-red commented:
* **#13382** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/13382?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/13382?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=13382).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Aug 29, 5:23 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Aug 29, 5:23 AM UTC**: `Boshen` added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc).

sapphi-red committed:
chore(codegen): skip stacktrace tests on big endian test workflow (#13382)

This PR tries to fix the following test failure.
https://github.com/oxc-project/oxc/actions/runs/17313194902/job/49151067079#step:5:325

This failure is probably happening because node binary is missing in the docker image that the test is running. While we can add the node binary in the docker container, I think we can skip the test as I guess there aren't any endian-specific code.
